cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/HBaseDDLExecutorFactory.java

```diff
@@ -46,7 +48,13 @@ public HBaseDDLExecutor get() {
       // HBase DDL executor extension is provided.
       executor = extensions.values().iterator().next();
     } else {
-      // HBase DDL executor extension is not provided. Return the version specific executor instance.
+      if (extensionDir != null) {
+        // Extension directory is provided but the extension is not loaded
+        throw new RuntimeException(String.format("HBaseDDLExecutor extension cannot be loaded from directory '%s'." +
+                                                   " Please make sure jar is available at that location with " +
```

Comment:
nit: for consistency, but the space at the end of the previous line instead of at the start of this line.